pkg/operator/hive/hive.go

```diff
@@ -36,6 +37,10 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+const (
+	dnsServersEnvVar = "ZONE_CHECK_DNS_SERVERS"
```

Comment:
Note: the env var is redeclared here to avoid circular import references. Ideally this is temporary and if it resolves the DNSZone issue, we'll make it a part of the HiveConfig type.